unitary/examples/quantum_chinese_chess/enums.py

```diff
@@ -25,6 +25,11 @@ class SquareState(enum.Enum):
     OCCUPIED = 1
 
 
+class GameState(enum.Enum):
```

Comment:
Again, I would just add this when you add the rule check.

Reply:
Done. Thanks.